src/System.Net.Http/tests/UnitTests/System.Net.Http.Unit.Tests.csproj

```diff
@@ -14,7 +14,6 @@
     <StringResourcesPath>../../src/Resources/Strings.resx</StringResourcesPath>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
-  
```

Comment:
It'd be good to revert all of the formatting changes, leaving just your one-line addition.